index.js

```diff
@@ -129,6 +129,9 @@ async function register (server, options) {
     }
 
     const info = request.info
+    const childBindings = getChildBindings(request)
+    request.logger = request.logger || logger.child(childBindings)
```

Comment:
Would you mind to put this in a if-block instead of using coditionals? I would like to avoid the assignment and calling `getChildBindings` if there is already a logger

Reply:
Done!